pipelines/pingcap/ticdc/latest/pull_cdc_kafka_integration_heavy_next_gen/pipeline.groovy

```diff
@@ -0,0 +1,181 @@
+// REF: https://www.jenkins.io/doc/book/pipeline/syntax/#declarative-pipeline
+// Keep small than 400 lines: https://issues.jenkins.io/browse/JENKINS-37984
+// should triggerd for master branches
+@Library('tipipeline') _
+
+final K8S_NAMESPACE = "jenkins-tiflow"
+final GIT_FULL_REPO_NAME = 'pingcap/ticdc'
+final GIT_CREDENTIALS_ID = 'github-sre-bot-ssh'
+final POD_TEMPLATE_FILE = 'pipelines/${GIT_FULL_REPO_NAME}/${BRANCH_ALIAS}/${JOB_BASE_NAME}/pod.yaml'
+final REFS = readJSON(text: params.JOB_SPEC).refs
+
+final tidbBranch = "master"
+final pdBranch = "master"
+final tikvBranch = "dedicated"
+final tiflashBranch = "master"
```

Comment:
keep the naming and style same as the tidb pipeline.groovy as I provided in previous reviews.

```
final TARGET_BRANCH_PD = (REFS.base_ref ==~ /release-.*/ ? REFS.base_ref : "master")
final TARGET_BRANCH_TIDB = (REFS.base_ref ==~ /release-.*/ ? REFS.base_ref : "master")
final TARGET_BRANCH_TIFLASH = (REFS.base_ref ==~ /release-.*/ ? REFS.base_ref : "master")
final TARGET_BRANCH_TIKV = (REFS.base_ref ==~ /release-.*/ ? REFS.base_ref : "dedicated")
```

Reply:
maybe we will create release-nextgen-* branches on ticdc repo in future